### PR TITLE
♻️dynamic-sidecar log-level review  (⚠️ devops)

### PR DESCRIPTION
**Pull request by GitHK**
## ⚠️ devops

Make sure `DYNAMIC_SIDECAR_LOG_LEVEL` is not set on the director-v2, if set put it to `INFO`.

## What do these changes do?

Review log levels in dynamic-sidecar. Making its logs more helpful.

## Related issue/s

- https://github.com/ITISFoundation/osparc-issues/issues/638

### CODE REVIEW

**Review by mrnicegyu11 — APPROVED** [review]
This will effectively remove some current logs, which might help to debug some issues down the road. Maybe keep this on hold for now? Anyway, you know best, I'm fine with the changes :) 

Proceed as you see fit! thx

File: services/director-v2/src/simcore_service_director_v2/core/settings.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ class DynamicSidecarSettings(BaseCustomSettings):
     )
 
     DYNAMIC_SIDECAR_LOG_LEVEL: str = Field(
-        "WARNING",
+        "INFO",
```
Comment (sanderegg) [inline services/director-v2/src/simcore_service_director_v2/core/settings.py:158]: so I think this is how I would suggest proceeding:
- let this to warning
- just change the INFO to WARNING directly in the deployment repositories

**Review by sanderegg — APPROVED** [review]

**Comment by GitHK** [comment]
> This will effectively remove some current logs, which might help to debug some issues down the road. Maybe keep this on hold for now? Anyway, you know best, I'm fine with the changes :)
> 
> Proceed as you see fit! thx

Actually it will not remove any logs, since they were never used. They were also incorrectly cataloged as info instead of debug.

File: services/director-v2/src/simcore_service_director_v2/core/settings.py (hunk 1 above)
Comment (GitHK) [reply services/director-v2/src/simcore_service_director_v2/core/settings.py:158]: OK, let's do it like you suggest. I will think the devops notes are sufficient. for this

**Pull request merged**